core/templates/dev/head/domain/objects/FractionObjectFactorySpec.js

```diff
@@ -74,37 +74,37 @@ describe('FractionObjectFactory', function() {
 
     it('should throw errors for invalid fractions', function() {
       // Invalid characters.
-      expect(function() {Fraction.fromRawInputString('3 \ b')}).toThrow(
+      expect(function() {Fraction.fromRawInputString('3 \ b');}).toThrow(
```

Comment:
@seanlip, We have options with `semi`
[`"omitLastInOneLineBlock": true`](https://eslint.org/docs/rules/semi#options).
(As semi-colon is not looking great in this case!)

Reply:
Let's do:
```
      expect(function() {
        Fraction.fromRawInputString('3 \ b');
      }).toThrow(
```

Ditto below.

Reply:
Okay! I'll make the necessary changes! Thanks!